src/algorithms/extract_containing_graph.hpp

```diff
@@ -19,21 +19,22 @@
 namespace vg {
 namespace algorithms {
 
-    /// Fills graph g with the subgraph of the handle graph source that contains all of the positions in
-    /// the positions vector and all other nodes and edges that can be reached within a maximum
-    /// distance from any of these positions. Node IDs in the subgraph are retained from the full graph.
+    /// Fills graph 'into' with the subgraph of the handle graph 'source' that contains all of the
+    /// positions in the positions vector and all other nodes and edges that can be reached within
+    /// a maximum distance from any of these positions. Node IDs in the subgraph are retained from
+    /// the source graph.
     ///
     /// Args:
     ///  vg                         graph to extract subgraph from
     ///  g                          graph to extract into
```

Comment:
The arguments have changed names, but this little table hasn't been updated.

Also, shouldn't it be `source` and `dest` and not `source` and `into`?